client/src/admin/adminActions.js

```diff
@@ -30,4 +35,22 @@ export function postAlert(authToken, alert) {
   });
 }
 
-export default postAlert;
+export function fetchHistory(accessToken) {
+  const headers = new Headers();
+  headers.append('Authorization', `Bearer ${accessToken}`);
+
+  const fetchInit = {
+    method: 'GET',
+    headers,
+  };
+
+  return dispatch => fetch('/api/alert/history', fetchInit)
+  .then(actionHelpers.checkStatus)
+  .then(actionHelpers.parseJSON)
+  .then((response) => {
+    dispatch(saveHistory(response));
+  })
+  .catch((error) => {
+    error.log(error);
```

Comment:
We need to display an error to the user with the banner if something goes wrong.